src/qml/IdlePauseRangePicker.qml

```diff
@@ -46,15 +48,14 @@ Column {
             leftMargin: Kirigami.Units.largeSpacing;
         }
 
-        first.onValueChanged: {
+        first.onMoved: {
```

Comment:
This was introduced in QQC2 2.5, and this file only asks for 2.4... Please make sure you import the right version :)